src/test/kotlin/no/nav/syfo/melding/kafka/KafkaLegeerklaringFraBehandlerConsumerSpek.kt

```diff
@@ -0,0 +1,102 @@
+package no.nav.syfo.melding.kafka
+
+import io.ktor.server.testing.*
+import io.mockk.*
+import kotlinx.coroutines.runBlocking
+import no.nav.syfo.melding.database.*
+import no.nav.syfo.melding.domain.MeldingType
+import no.nav.syfo.melding.kafka.legeerklaring.KafkaLegeerklaringConsumer
+import no.nav.syfo.melding.kafka.legeerklaring.LEGEERKLARING_TOPIC
+import no.nav.syfo.testhelper.*
+import no.nav.syfo.testhelper.generator.*
+import no.nav.syfo.testhelper.mock.mockKafkaConsumer
+import org.amshove.kluent.shouldBe
+import org.amshove.kluent.shouldBeEqualTo
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import java.util.*
+
+class KafkaLegeerklaringFraBehandlerConsumerSpek : Spek({
+
+    with(TestApplicationEngine()) {
+        start()
+        val externalMockEnvironment = ExternalMockEnvironment.instance
+        val database = externalMockEnvironment.database
+
+        val personIdent = UserConstants.ARBEIDSTAKER_PERSONIDENT
+        val behandlerPersonIdent = UserConstants.BEHANDLER_PERSONIDENT
+        val behandlerNavn = UserConstants.BEHANDLER_NAVN
+
+        afterEachTest {
+            database.dropData()
+        }
+
+        val kafkaLegeerklaringConsumer = KafkaLegeerklaringConsumer(
+            database = database,
+        )
+
+        describe("Read legeerklaring sent from behandler to NAV from Kafka Topic") {
+            describe("Happy path") {
+                it("Receive legerklaring") {
```

Comment:
Kunne kanskje skrevet noe i testnavnet at man ikke lagrer pga manglende match conversationRef?
Jeg forventet at denne skulle lagre noe i databasen bare basert på navnet, mener at den første testen i happy path kanskje burde være "default-success"-casen 🤔

Reply:
Ja, ser Anders kommenterte det samme

Reply:
Testnavnet kan gjerne si noe om ønsket oppførsel også (at vi ikke lagrer). Og kanskje et case for manglende conversationRef og et case for ukjent conversationRef.

Reply:
Ja enig, skal fikse.